src/components/Links/ContentHashLink.js

```diff
@@ -39,7 +39,7 @@ const ContentHashLink = ({ value, contentType }) => {
     return <DecodedError>{error}</DecodedError>
   }
   if (protocolType === 'ipfs') {
```

Comment:
`https://dweb.link/ipfs/${decoded}` will also work (will redirect to correct subdomain + convert CID to DNS-compatible encoding, if needed)